src/view/com/composer/char-progress/CharProgress.tsx

```diff
@@ -27,35 +26,51 @@ export function CharProgress({
   const pal = usePalette('default')
   const textColor = count > maxLength ? '#e60000' : pal.colors.text
   const circleColor = count > maxLength ? '#e60000' : pal.colors.link
+  const remainingCount = maxLength - count
+  const shouldShowText = remainingCount <= 20
+  const shouldShowCircle = remainingCount > -10
```

Comment:
Where are you getting 20 and -10 from? Are these experimentally computed?

Reply:
Also, is there ever a possibility that both `shouldShowText` and `shouldShowCircle` will be true? Will the UI elements conflict in that case? Would it be more sensible to use a single var?

Reply:
Yes, `shouldShowText` and `shouldShowCircle` can be true at the same time when `remainingCount > -10 && remainingCount <= 20`, and that's when we display the number at the center of the progress circle.

Creating two variables was intentional, and I can not think of a way to use a single variable.

20 and -10 are from X